pkg/scheduler/apis/config/v1beta1/defaults.go

```diff
@@ -273,6 +273,18 @@ func SetDefaults_VolumeBindingArgs(obj *v1beta1.VolumeBindingArgs) {
 	if obj.BindTimeoutSeconds == nil {
 		obj.BindTimeoutSeconds = pointer.Int64Ptr(600)
 	}
```

Comment:
I see there is a v1 type. Do we need to add defaults there? I notice there are no defaults at all for v1. cc @ahg-g

Reply:
the v1 is not for component config, it is for the old policy api, which we will be deprecating in the next release.